src/styles/ForecastSummaries.css

```diff
@@ -0,0 +1,14 @@
+.forecast-summaries {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.icon .description .temperature p {
+  justify-content: center;
+  text-align: center;
+}
```

Comment:
Not only is this selector way too specific, it isnt getting used anywhere! You should remove this. targeting elements its always best to keep the specificity down to the lowest level you can, ideally targeting them directly and over-riding really specific rules like this gets harder the more specific you make the selector